src/Summoner/CLI.hs

```diff
@@ -31,27 +36,52 @@ summon = execParser prsr >>= runWithOptions
 
 -- | Run 'hs-init' with cli options
 runWithOptions :: InitOpts -> IO ()
-runWithOptions (InitOpts projectName targets) = do
-     -- Generate the project.
-    generateProject projectName targets
+runWithOptions (InitOpts projectName cliConfig maybeFile) = do
+    file <- case maybeFile of
+        Nothing -> (</> "summoner.toml") <$> getHomeDirectory
+        Just x  -> pure x
+    isFile <- doesFileExist file
+    fileConfig <-
+        if isFile
+        then loadFileConfig file
+        else pure mempty
```

Comment:
After thinking a little bit more on this, I came to decision that logic here should be slightly complicated to increase UX.

1. If file exist, print information message (probably we can use `Blue` for information?) that this file configuration will be used.
2. If this is a default file `~/summoner.toml` and it doesn't exist then print info message that no configuration file was specified (or will be used).
3. If this is not a default configuration file (i.e. specified by `--file`) then we should abort immediately because I think that if you specify file manually through CLI you expect this file to exist. But if you have typo in file name, it's better to see error that such file doesn't exist and fix CLI command instead of silently continue to work.